devfile.yaml

```diff
@@ -0,0 +1,54 @@
+commands:
```

Comment:
what it is good for?

Reply:
Used for developing directly on [openshift](https://marketplace.visualstudio.com/items?itemName=redhat.vscode-openshift-connector).